### PR TITLE
Move custom polyfills to the server bundle

### PR DESCRIPTION
**Pull request by ChristophWurst**
This moves this ugly code somewhere it doesn't bother us anymore ;)

### CODE REVIEW

File: core/src/Polyfill/windows-phone.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// fix device width on windows phone
+if ("-ms-user-select" in document.documentElement.style && navigator.userAgent.match(/IEMobile\/10\.0/)) {
```
Comment (rullzer) [inline core/src/Polyfill/windows-phone.js]: Can we deprecate this? IEMobile 10 is over 6 years old :see_no_evil:

File: core/src/Polyfill/console.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+if (typeof console === "undefined" || typeof console.log === "undefined") {
```
Comment (rullzer) [inline core/src/Polyfill/console.js]: Do we have knowledge of any browser that doesn't have a console?

Comment (ChristophWurst) [reply core/src/Polyfill/console.js]: IE mobile 10? 🙈

Comment (ChristophWurst) [reply core/src/Polyfill/console.js]: https://caniuse.com/#search=console

Comment (ChristophWurst) [reply core/src/Polyfill/console.js]: As discussed with @skjnldsv at the contacts app we should finally come up with an exact list of browser we do support.

File: core/src/Polyfill/windows-phone.js (hunk 1 above)
Comment (ChristophWurst) [reply core/src/Polyfill/windows-phone.js]: We can just add it to the deprecation notice and remove that with 17

**Review by rullzer — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]
I'm okay since this is a migration and not a change, though we really should adress the comments here. Shall we do this here?

**Comment by ChristophWurst** [comment]
No, please let's keep refactorings and actual changes separate.

**Pull request merged**